config/routes.rb

```diff
@@ -2,5 +2,6 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
-end
+resources :users, only: [:index, :show] do
+  resources :posts, only: [:index, :show]
+end
```

Comment:
[OPTIONAL]: kindly you need to add one more 'end' to the routes file

Reply:
@Danieldotcomcoder, I have implemented that change, Thanks for your kind review and feedback.